tests/integration_tests/substrate/vanchorTests.spec.ts

```diff
@@ -132,7 +138,7 @@ async function createVAnchorWithDeposit(
     extAmount: extAmount.toString(),
     fee: fee.toString()
   };
-  const data = await provingManager.prove('vanchor', setup);
+  const data = await provingManager.prove('vanchor', setup) as VAnchorProof;
```

Comment:
I'm not sure how to get the typed output from proving manager without explicitly casting as `VAnchorProof`

Reply:
This is OK.